low-level/files/opt/seagate/sspl/prerequisites/sspl_dev_deploy.sh

```
@@ -0,0 +1,484 @@
+disable_sub_mgr=false
+skip_bmc=false
+initialize_sspl=false
+TARGET_BUILD=
+RPMS_PATH=
+COMPONENT="SSPL"
+PRODUCT_VERSION="LDR_R2"
+NODE="srvnode-1"
+RMQ_USER="sspluser"
+RMQ_PASSWD=""
+CNTRLR_A="10.0.0.2"
+CNTRLR_B="10.0.0.3"
+CNTRLR_A_PORT="80"
+CNTRLR_B_PORT="80"
+CNTRLR_USER=
+CNTRLR_PASSWD=
+BMC_IP=""
+BMC_USER=""
+BMC_PASSWD=""
+
+
+usage()
+{
+    echo "\
+    SSPL prerequisite script.
+    (Bounded to single node provisioning)
+
+    Usage:
+         $0
+            [-V|--product_version  <LDR_R2>]
+            [-N|--node  <Node name/id>]
+            [-A|--cntrlr_a  <controller A IP>]
+            [-B|--cntrlr_b  <controller B IP>]
+            [-L|--local_rpms_path   <sspl rpms location>]
+            [-T|--target_build_url  <target build url>]
+            [--Ap|--cntrlr_a_port  <controller A Port>]
+            [--Bp|--cntrlr_b_port  <controller B Port>]
+            [-U|--cntrlr_user   <username>]
+            [-P|--cntrlr_pass   <password>]
+            [--i|--bmc_ip     <bmc ipv4 address>]
+            [--u|--bmc_user   <bmc user>]
+            [--p|--bmc_passwd <bmc password>]
+            [--Ru|--rmq_user    <rabbitmq username>]
+            [--Rp|--rmq_passwd  <rabbitmq password>]
+            [--St|--storage_type  <storage type>]
+            [--Sr|--server_type    <server type>]
+            [-D|--disable-sub-mgr]
+            [--standalone_installation]
+            [--initialize_sspl]
+            [--setup_repo]
+            [--cleanup]
+            [-h|--help]
+
+    OPTION:
+    -V      <PRODUCT VERSION>   Product version (Default 'LDR_R2')
+    -N      <NODE NAME>         Default 'srvnode-1'
+    -A      <IP ADDRESS>        IP address of controller A (default '10.0.0.2')
+    -B      <IP ADDRESS>        IP address of controller B (default '10.0.0.3')
+    --Ap    <CNTRLR A PORT>     Controller A port (default '80')
+    --Bp    <CNTRLR A PORT>     Controller B port (default '80')
+    -U      <USER NAME>         Username for controller
```

defaults for all, except passwd can be kept

---

Okay.